src/rng.jl

```diff
@@ -0,0 +1,104 @@
+using Random123
+using Random
+using Distributions
+
+import Base.rand
+import Random.seed!
+import Random123: set_counter!
```

Comment:
IMO it is safer and more readable to not import the extended functions but to define the new methods with
```julia
Base.rand(...) = ...
```
etc.